front/app/containers/Admin/formBuilder/components/FormBuilderSettings/index.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { useState, useEffect } from 'react';
+
+// styles
+import { colors } from 'utils/styleUtils';
+
+// components
+import {
+  Box,
+  Toggle,
+  Text,
+  stylingConsts,
+} from '@citizenlab/cl2-component-library';
+import Button from 'components/UI/Button';
+import InputMultilocWithLocaleSwitcher from 'components/UI/InputMultilocWithLocaleSwitcher';
+import { SectionField, SectionTitle } from 'components/admin/Section';
+import CloseIconButton from 'components/UI/CloseIconButton';
+
+// intl
+import messages from '../messages';
+import { FormattedMessage } from 'utils/cl-intl';
+
+// Types
+import { Multiloc } from 'typings';
+import {
+  IFlatCustomField,
+  IFlatUpdateCustomField,
+} from 'services/surveyCustomFields';
+
+// utils
+import { isNilOrError } from 'utils/helperUtils';
+
+interface Props {
+  field?: IFlatCustomField;
```

Comment:
Making this property optional means that we have to take the possible absence of `field` into account in several places in this file, which makes the code more complex than needed. From a conversation with Edwin and Amanda on Slack:
> The alternative is to move the check at the point the component is rendered. Then we would only render it if it is provided. It might even be better.